lib/msal-node/test/client/ClientCredentialClient.spec.ts

```diff
@@ -401,23 +382,79 @@ describe("ClientCredentialClient unit tests", () => {
         checkMockedNetworkRequest(returnVal, checks);
     });
 
-    it("Does not add claims when empty object provided", async () => {
-        sinon
-            .stub(Authority.prototype, <any>"getEndpointMetadataFromNetwork")
-            .resolves(DEFAULT_OPENID_CONFIG_RESPONSE.body);
-        sinon
-            .stub(
-                ClientCredentialClient.prototype,
-                <any>"executePostToTokenEndpoint"
-            )
-            .resolves(CONFIDENTIAL_CLIENT_AUTHENTICATION_RESULT);
+    describe("CAE, claims and client capabilities", () => {
```

Comment:
This test is the only change in this file that needs to be reviewed. The rest of the changes are improvements to existing tests.